neuro_extras/main.py

```diff
@@ -221,6 +252,182 @@ def image() -> None:
     pass
 
 
+@main.group()
+def data() -> None:
+    pass
+
+
+class UrlType(Enum):
+    UNSUPPORTED = 0
+    LOCAL = 1
+    CLOUD = 2
+    STORAGE = 3
+
+    @staticmethod
+    def get_type(url: URL) -> "UrlType":
+        if url.scheme == "storage":
+            return UrlType.STORAGE
+        if url.scheme == "":
+            return UrlType.LOCAL
+        if url.scheme in ("s3", "gs"):
+            return UrlType.CLOUD
+        return UrlType.UNSUPPORTED
+
+
+async def _data_cp(source: str, destination: str, extract: bool) -> None:
+    source_url = URL(source)
+    destination_url = URL(destination)
```

Comment:
Tests fail on Windows:
```destination = 'C:\\Users\\runneradmin\\.neuro\\tmp\\tmpe1ud6v9p', extract = True
...
E           ValueError: Unsupported destination URL scheme: c```
Could we re-use neuro API's parsing utilities? for example:
https://github.com/neuromation/platform-client-python/blob/562d170b3ecfb14513779a072f27c218232adaa4/neuromation/api/url_utils.py#L14

Reply:
True, we could use it, I've tried it on my unix and win machines neuromation's `uri_from_cli` function vs yarl's `URL constructor`.
neuromation's uri_from_cli is more robust, let's use it.